src/main/java/de/samply/share/broker/rest/InquiryHandler.java

```diff
@@ -127,9 +123,9 @@ static boolean isQueryLanguageCql(String queryLanguage) {
    */
   public int storeAndRelease(String simpleQueryDtoJson, int userid, String inquiryName,
       String inquiryDescription, int exposeId, int voteId, List<String> resultTypes,
-      boolean bypassExamination, boolean isMonitoring) {
+      boolean bypassExamination, boolean isMonitoring, boolean structuredQuery) {
```

Comment:
Please don't add another boolean toggle. Use a query type instead and remove also isMonitoring.